src/main/java/com/uber/cadence/internal/sync/SyncDecisionContext.java

```diff
@@ -412,6 +412,8 @@ private ExecuteLocalActivityParameters constructExecuteLocalActivityParameters(
             .setWorkflowIdReusePolicy(options.getWorkflowIdReusePolicy())
             .setRetryParameters(retryParameters)
             .setCronSchedule(options.getCronSchedule())
+            .setMemo(options.getMemo())
```

Comment:
look at line 387. I think we also need to do the same thing

Reply:
updated